entity-service/src/main/kotlin/com/egm/stellio/entity/repository/EntityRepository.kt

```diff
@@ -8,36 +8,32 @@ import org.springframework.stereotype.Repository
 @Repository
 interface EntityRepository : Neo4jRepository<Entity, String> {
 
-    @Query("""
-        MATCH (entity:Entity { id: {id} })
-        RETURN entity
-    """)
+    @Query("MATCH (entity:Entity { id: \$id })" +
```

Comment:
You cannot do multi-line strings anymore?

Reply:
From the documentation:
A raw string is delimited by a triple quote ("""), contains no escaping and can contain newlines and any other character.
So we need to use the standard string.

Or, there is another option (use the literal representation), for this example:
"""MATCH (entity:Entity { id: ${'$'}id }) RETURN entity"""